Vazaar.Core.Api/Controllers/TokenController.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Text;
+using Vazaar.Core.Membership.Entities;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.IdentityModel.Tokens;
+using Microsoft.Extensions.Configuration;
+using System.Threading.Tasks;
+using System;
+using System.Linq;
+
+namespace Vazaar.Core.Api.Controllers
+{
+    [Route("api/v1/[controller]")]
+    [ApiController]
+    public class TokenController : ControllerBase
+    {
+        private readonly IConfiguration _configuration;
+        private readonly SignInManager<ApplicationUser> _signInManager;
+        private readonly UserManager<ApplicationUser> _userManager;
```

Comment:
Please declare private class members according to our C# coding standard:
https://github.com/hassanhabib/CSharpCodingStandard